### PR TITLE
FTS for new Global Search endpoint

### PR DESCRIPTION
**Pull request by padmapriyanalam**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RDM-12764

### Change description ###

FTS for new Global Search endpoint

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/aat/resources/features/F-1005 - GlobalSearch - Search cases/F-1005_CreateCasePreRequisiteCaseworker.td.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+	"_guid_": "F-1005_CreateCasePreRequisiteCaseworker",
+	"_extends_": "F-1004_CreateCasePreRequisiteCaseworker"
```
Comment (sullivangareth) [inline src/aat/resources/features/F-1005 - GlobalSearch - Search cases/F-1005_CreateCasePreRequisiteCaseworker.td.json:3]: Bit picky - but if referring to something in F-1004 from F-1005, should it be in a common location, that can then be referenced from F-1004 & F-1005 ?

File: src/aat/resources/features/F-1005 - GlobalSearch - Search cases/S-1005.3.td.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    "body": {
+      "searchCriteria": {
+      }
+    }
+  },
+  "expectedResponse": {
+    "_extends_": "Common_400_Response",
+    "body" : {
+      "exception" : "org.springframework.web.bind.MethodArgumentNotValidException",
+      "timestamp" : "[[ANYTHING_PRESENT]]",
```
Comment (sullivangareth) [inline src/aat/resources/features/F-1005 - GlobalSearch - Search cases/S-1005.3.td.json:22]: I think you can remove this one as it comes from `Common_4XX_Response_Body`

File: src/aat/resources/features/F-1005 - GlobalSearch - Search cases/S-1005.1.td.json [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    "body": {
+      "searchCriteria": {
+        "caseReferences": [
+          "${[scenarioContext][childContexts][F-1005_CreateCasePreRequisiteCaseworker][testData][actualResponse][body][id]}"
+        ]
+      }
+    }
+  },
+  "expectedResponse": {
+    "_extends_": "Common_200_Response",
```
Comment (sullivangareth) [inline src/aat/resources/features/F-1005 - GlobalSearch - Search cases/S-1005.1.td.json:21]: Could create a `Global_Search_200_Response` containing body

```
"resultInfo": {
        "casesReturned": 1,
        "caseStartRecord": 1,
        "moreResultsToGo": false
      }
```
      
which you can then override where needed ?

File: src/aat/resources/features/F-1005 - GlobalSearch - Search cases/F-1005_UpdateCasePreRequisiteCaseworker.td.json [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+        "Address": {
+          "AddressLine1": "updated Address"
+        }
+      },
+      "event_token": "${[scenarioContext][siblingContexts][F-1005_GetCaseworkerUpdateToken][testData][actualResponse][body][token]}"
+    }
+  },
+
+  "expectedResponse": {
+    "responseCode": 201,
```
Comment (sullivangareth) [inline src/aat/resources/features/F-1005 - GlobalSearch - Search cases/F-1005_UpdateCasePreRequisiteCaseworker.td.json:24]: "_extends_": "Common_201_Response"

File: src/aat/resources/features/F-1005 - GlobalSearch - Search cases/S-1005.5.td.json [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+    "body": {
+      "searchCriteria": {
+        "CCDCaseTypeIds": [
+          "FT_GlobalSearch"
+        ]
+      }
+    }
+  },
+  "expectedResponse": {
+    "_extends_": "Common_200_Response",
```
Comment (sullivangareth) [inline src/aat/resources/features/F-1005 - GlobalSearch - Search cases/S-1005.5.td.json:28]: Same comment as previously re: 

Global_Search_200_Response

**Review by sullivangareth — APPROVED** [review]

File: src/aat/resources/features/F-1005 - GlobalSearch - Search cases/F-1005_UpdateCasePreRequisiteCaseworker.td.json (hunk 4 above)
Comment (marianadpereira) [reply src/aat/resources/features/F-1005 - GlobalSearch - Search cases/F-1005_UpdateCasePreRequisiteCaseworker.td.json:24]: done 😊

File: src/aat/resources/features/F-1005 - GlobalSearch - Search cases/F-1005_CreateCasePreRequisiteCaseworker.td.json (hunk 1 above)
Comment (marianadpereira) [reply src/aat/resources/features/F-1005 - GlobalSearch - Search cases/F-1005_CreateCasePreRequisiteCaseworker.td.json:3]: Hi Gareth, doesn't make sense to have a file for case creation for a specific case type under the common files, as it is used only for these 2 features. I can either leave it as it is or I can not extend and add all the same information to the F-1005 file.

File: src/aat/resources/features/F-1005 - GlobalSearch - Search cases/S-1005.3.td.json (hunk 2 above)
Comment (marianadpereira) [reply src/aat/resources/features/F-1005 - GlobalSearch - Search cases/S-1005.3.td.json:22]: done 😊

**Pull request merged**